### ISSUE TITLE
could we have access to the training code now?

### ISSUE THREAD

**Issue by chenxinli001** [willprice/epic-kitchens-activity-stories#1]
could we have access to the training code now? i'm really interested in this research but there are only labels released. Thanks

**Comment by vineetparikh** [comment]
In addition, per the paper it looks like the labels were collected partially via self-supervision and partially via manual annotation when training. Are the labels here mostly self-supervised or mostly manually annotated and/or what's the split between the two (for each of the train/val/test splits)?

**Comment by dimadamen** [comment]
The labels here are ONLY manually collected. All the train/val/test here is the manual ground-truth collected from manual annotations.
The self-supervised task is randomly done and varies per run, and so no labels are provided for these, as these are done on the fly.
